### ISSUE TITLE
Mark plugin @threadSafe to comply with Maven 3.5.0 parallel execution

### ISSUE THREAD

**Issue by Emontis** [jdcasey/directory-maven-plugin#7]
When executing a parallel build with a directory-maven-plugin execution, Maven 3.5.0 will output a warning

```
[WARNING] *****************************************************************
[WARNING] * Your build is requesting parallel execution, but project      *
[WARNING] * contains the following plugin(s) that have goals not marked   *
[WARNING] * as @threadSafe to support parallel building.                  *
[WARNING] * While this /may/ work fine, please look for plugin updates    *
[WARNING] * and/or request plugins be made thread-safe.                   *
[WARNING] * If reporting an issue, report it against the plugin in        *
[WARNING] * question, not against maven-core                              *
[WARNING] *****************************************************************
[WARNING] The following plugins are not marked @threadSafe in module:
[WARNING] org.commonjava.maven.plugins:directory-maven-plugin:0.1
[WARNING] Enable debug to see more precisely which goals are not marked @threadSafe.
[WARNING] *****************************************************************
```

Can you add this please? Thank you very much.

**Comment by Emontis** [comment]
Does not work with release 0.3 for highest basedir goal.

**Comment by turbanoff** [comment]
@Emontis yep( I've fixed in #13. Wait for merge

**Comment by Emontis** [comment]
Okay thanks, thought it was fixed with #12 and included in release 0.3.
